docs/IE_DG/Extensibility_DG/Custom_ONNX_Ops.md

```diff
@@ -1,7 +1,7 @@
 # Custom ONNX* Operators {#openvino_docs_IE_DG_Extensibility_DG_Custom_ONNX_Ops}
 
 The ONNX\* importer provides a mechanism to register custom ONNX operators based on predefined or custom nGraph operations.
-The function responsible for registering a new operator is called `ngraph::onnx_import::register_operator` and defined in the `onnx_import/onnx_utils.hpp` file.
+The function responsible for registering a new operator is called `ov::onnx_import::register_operator` and defined in the `onnx_import/onnx_utils.hpp` file.
```

Comment:
I think `onnx_import::` is a legacy low level API.
I think we shouldn't add it to new API.
new API has an extension mechanism. please keep old namespace for legacy importer API